packages/ember-views/lib/views/view.js

```diff
@@ -1214,48 +1215,36 @@ var View = CoreView.extend({
     @private
   */
   _applyAttributeBindings: function(buffer, attributeBindings) {
-    var attributeValue;
     var unspecifiedAttributeBindings = this._unspecifiedAttributeBindings = this._unspecifiedAttributeBindings || {};
 
-    forEach(attributeBindings, function(binding) {
-      var split = binding.split(':');
-      var property = split[0];
-      var attributeName = split[1] || property;
+    var binding, split, property, attrName, attrNode, attrValue, value;
+    var i, l;
+    for (i=0, l=attributeBindings.length; i<l; i++) {
+      binding = attributeBindings[i];
+      split = binding.split(':');
+      property = split[0];
+      attrName = split[1] || property;
```

Comment:
Oof this is for micro-syntax not for namespaced attrs. The solution is not as super-simple as it seems, imo.

* `isDisabled:disabled` is a mapping of `view-prop:attr`
* `xlink:href` is a namespaced attr
* `hrefLink:xlink:href` is a mapping of `view-prop:namespaced-attr`

There isn't really a way to support all that. I would propose:

* `isDisabled:disabled` is a mapping of `view-prop:attr`
* `xlink:href` is a mapping of `view-prop:attr` (gotcha)
* `hrefLink:xlink:href` is a mapping of `view-prop:namespaced-attr`

This would be consistent. Basically, the micro-syntax sucks. Maybe we can drop it in 2.0. I don't want to tackle it here though.